main.js

```diff
@@ -0,0 +1,196 @@
+    if (e.key === 'Enter') {
+        const guestInput = inputField.value;
+        // opdracht 9
+        try {
+            const response = await axios.get(
+                "https://restcountries.eu/rest/v2/name/" + guestInput + "?fullText=true"
+            );
+            const countryName = response.data[0].name;
+            const subRegion = response.data[0].subregion;
+            const population = response.data[0].population;
+            const capital = response.data[0].capital;
+            const valuta = response.data[0].currencies;
+            const valutaName = valuta.map((currencie) => {
+                return currencie.name;
+            })
+
+            function makevalutaString() {
+                switch (valutaName.length) {
+                    case 1:
+                        return "and you can pay with " + valutaName[0]+".";
+                    case 2:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1]+".";
+                    case 3:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1] + ", " + valutaName[2]+".";
+                    case 4:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1] + ", " + valutaName[2] + ", " + valutaName[3]+".";
+                    case 5:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1] + ", " + valutaName[2] + ", " + valutaName[3] + ", " + valutaName[4]+".";
+                    case 6:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1] + ", " + valutaName[2] + ", " + valutaName[3] + ", " + valutaName[4] + ", " + valutaName[5]+".";
+                    case 7:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1] + ", " + valutaName[2] + ", " + valutaName[3] + ", " + valutaName[4] + ", " + valutaName[5] + ", " + valutaName[6]+".";
+                    case 8:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1] + ", " + valutaName[2] + ", " + valutaName[3] + ", " + valutaName[4] + ", " + valutaName[5] + ", " + valutaName[6] + ", " + valutaName[7]+".";
+                    case 9:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1] + ", " + valutaName[2] + ", " + valutaName[3] + ", " + valutaName[4] + ", " + valutaName[5] + ", " + valutaName[6] + ", " + valutaName[7] + ", " + valutaName[8]+".";
+                    case 10:
+                        return "and you can pay with " + valutaName[0] + ", " + valutaName[1] + ", " + valutaName[2] + ", " + valutaName[3] + ", " + valutaName[4] + ", " + valutaName[5] + ", " + valutaName[6] + ", " + valutaName[7] + ", " + valutaName[8] + ", " + valutaName[9]+".";
+                    default:
+                        return "Dat is raar geen valuta!! ";
+                }
```

Comment:
Ik vind dit niet alleen slecht onderhoudbaar, maar ook heel lelijk 😅  Je zou dit op kunnen lossen op zo'n manier:
`return "and you can pay with " + valutaName.join(", ") + ".";`